### ISSUE TITLE
[Installation] Deprecated Modules, Permissions and Renamed Modules

### ISSUE THREAD

**Issue by Bizarrus** [strongloop/strong-pm#411]
I wan't to try to install `strongloop`...

It's not "easy to install" like the documentation/website says..

**System**
```
Debian 8
node.js: v9.11.1
npm: 5.6.0
```

**Logfile**
```!cli
root@euve261743:/opt/robot/bin# npm install -g strongloop
npm WARN deprecated nodemailer@2.7.2: All versions below 4.0.1 of Nodemailer are deprecated. See https://nodemailer.com/status/
npm WARN deprecated mailcomposer@4.0.1: This project is unmaintained
npm WARN deprecated socks@1.1.9: If using 2.x branch, please upgrade to at least 2.1.6 to avoid a serious bug with socket data flow and an import issue introduced in 2.1.0
npm WARN deprecated buildmail@4.0.1: This project is unmaintained
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
npm WARN deprecated formatio@1.2.0: This package is unmaintained. Use @sinonjs/formatio instead
npm WARN deprecated ejs@1.0.0: Critical security bugs fixed in 2.5.5
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated natives@1.1.3: This module relies on Node.js's internals and will break at some point. Do not use it, and update to graceful-fs@4.x.
npm WARN deprecated sprintf@0.1.5: The sprintf package is deprecated in favor of sprintf-js.
npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
npm WARN deprecated connect@2.30.2: connect 2.x series is deprecated
npm WARN deprecated coffee-script@1.6.3: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
npm WARN deprecated sprintf@0.1.1: The sprintf package is deprecated in favor of sprintf-js.
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm ERR! path /usr/lib/node_modules/strongloop/node_modules/nodefly-register/node_modules/request/node_modules/aws-sign
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall rename
npm ERR! enoent ENOENT: no such file or directory, rename '/usr/lib/node_modules/strongloop/node_modules/nodefly-register/node_modules/request/node_modules/aws-sign' -> '/usr/lib/node_modules/strongloop/node_modules/nodefly-register/node_modules/request/node_modules/.aws-sign.DELETE'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-04-06T14_07_26_940Z-debug.log
root@euve261743:/opt/robot/bin#
```

**Comment by smartmouse** [comment]
You are using root to install?  have you tried npm i -g strongloop --unsafe-perm?  I do not have debain, I only use linux ubundtu

**Comment by jhenry9636** [comment]
Hi ,

Thanks for this wonderful utility however is it still intended to be used?

I'm seeing the same thing on my machine.

```
npm WARN deprecated nodemailer@2.7.2: All versions below 4.0.1 of Nodemailer are deprecated. See https://nodemailer.com/status/
npm WARN deprecated mailcomposer@4.0.1: This project is unmaintained
npm WARN deprecated socks@1.1.9: If using 2.x branch, please upgrade to at least 2.1.6 to avoid a serious bug with socket data
flow and an import issue introduced in 2.1.0
npm WARN deprecated buildmail@4.0.1: This project is unmaintained
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpj
s/gulp-util-ca3b1f9f9ac5
npm WARN deprecated ejs@1.0.0: Critical security bugs fixed in 2.5.5npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of N
ode.js
npm WARN deprecated sprintf@0.1.5: The sprintf package is deprecated in favor of sprintf-js.
npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
npm WARN deprecated connect@2.30.2: connect 2.x series is deprecated
npm WARN deprecated coffee-script@1.6.3: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
npm WARN deprecated sprintf@0.1.1: The sprintf package is deprecated in favor of sprintf-js.
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm ERR! path /usr/local/lib/node_modules/strongloop/node_modules/nodefly-register/node_modules/request/node_modules/aws-sign
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall rename
npm ERR! enoent ENOENT: no such file or directory, rename '/usr/local/lib/node_modules/strongloop/node_modules/nodefly-register/node_modules/request/node_modules/aws-sign' -> '/usr/local/lib/node_modules/strongloop/node_modules/nodefly-register/node_modules/request/node_modules/.aws-sign.DELETE'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jarradhenry/.npm/_logs/2018-06-14T23_09_06_001Z-debug.log
```


Can I still use it? I've been trying for 2 days sorting through errors.
